mantidproject/mantid
# Log averaging muon analysis

Within the muon analysis code the time averaging was incorrect. It has been altered to use the timeAverageValue function and if the running log is present then it is filtered accordingly. 

**To test: If the new time averaging is correct**

 
1. Select: Interfaces -> Muon ->Muon Analysis  and a new window should open. 
2. Load some muon data (e.g.  HIFI108376.nxs). 
3. Click the Data Analysis tab. 
4. Under functions  right click and select "add function". In the new box select Background then LinearBackground. 
5. Click the Fit button and select Fit from the options.
6.  Click the Results Table tab. 
7. Select the values that you want to investigate (I have found ICP_SYS_TD to be good). 
8. Click Create Table. 

9. To create the comparison data right click on MuonAnalysis in the workspaces and select Sample Logs...
10. A window should appear. Select the value that is being investigated. 
11. The Time Avg: box should contain the same number as the results table. 

12. It is worth checking that the results are different between Status + Period and None. The results should match Status + Period (which is the same as Status) if the running log is present. 


Fixes #18061.


Release notes: see [here](https://github.com/mantidproject/mantid/pull/18439/commits/816bcc69da686feebd4d7b74c0f24a0d7368bf90)
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
